### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by YaseenSaif**
adding another button belongs to yaseen

### CODE REVIEW

**Review by AbdulqaderAlhemyari — CHANGES_REQUESTED** [review]
you are professional programmer 
keep up

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -20,7 +20,19 @@
...
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
+        android:textSize="36sp"
+        android:id="@+id/tv_yaseen" />
 
+    <Button
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Git_yaseen"
+        android:id="@+id/GitBtn_yaseen">
```
Comment (YaseenSaif) [inline app/src/main/res/layout/activity_main.xml:35]: button added by yaseen

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -20,7 +20,19 @@
         android:layout_height="wrap_content"
         android:text="Git"
         android:id="@+id/GitBtn">
+    </Button>
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
+        android:textSize="36sp"
+        android:id="@+id/tv_yaseen" />
```
Comment (YaseenSaif) [inline app/src/main/res/layout/activity_main.xml:29]: text added by yaseen

File: app/src/main/java/com/example/test1git/MainActivity.kt [hunk 3]
```diff
@@ -7,15 +7,23 @@ import android.widget.TextView
 
 class MainActivity : AppCompatActivity() {
     lateinit var textView:TextView
+    lateinit var textViewY:TextView
     lateinit var btn:Button
```
Comment (AbdulqaderAlhemyari) [inline app/src/main/java/com/example/test1git/MainActivity.kt:11]: line added by yaseen

Comment (YaseenSaif) [reply app/src/main/java/com/example/test1git/MainActivity.kt:11]: how did tell u?

**Pull request merged**